config/locales/en.yml

```diff
@@ -2,13 +2,27 @@ en:
   activerecord:
     errors:
       models:
+        organisation:
+          attributes:
+            type:
+              blank: Select an organisation type
         provider:
           attributes:
             provider_code:
               taken: Provider already exists
+              blank: Provider name can't be blank
+        school:
+          attributes: 
+            urn: 
+              taken: School already exists
+              blank: School name can't be blank
+            gias_school:
+              required: School isn't recognised
```

Comment:
As discussed, error messages should instruct the user how to fix the problem and match the input on the field, eg: 
<img width="1001" alt="image" src="https://github.com/DFE-Digital/itt-mentor-services/assets/44073106/0bb82f28-15b5-4892-a286-4a59182821c2">

I know we didn't have designs for the school not being recognised or the school already being a placement school. I think we definitely need a steer from Simon on the unhappy path where a user chooses a school that is already a placement school -- would be good maybe to let them know earlier in the journey. 

But some suggestions for messages are:
```suggestion
              blank: Enter a school name, URN or postcode
            gias_school:
              required: Enter a valid school name, URN or postcode
```

Reply:
resolved in [Improve school form error handling](https://github.com/DFE-Digital/itt-mentor-services/pull/73/commits/ba6519cdcd2a9e194a2109f8b2b5ac1f233cb343)